hail/src/main/scala/is/hail/services/shuffler/server/ShuffleServer.scala

```diff
@@ -207,17 +208,20 @@ class Shuffle (
   }
 }
 
-class ShuffleServer (
-  ssl: SSLContext,
-  port: Int
-) extends AutoCloseable {
+object ShuffleServer {
+  def main(args: Array[String]): Unit =
+    using(new ShuffleServer())(_.serve())
+}
+
+class ShuffleServer() extends AutoCloseable {
+  val ssl = getSSLContext
+  val port = 443
   val log = Logger.getLogger(this.getClass.getName());
 
   val shuffles = new ConcurrentSkipListMap[Array[Byte], Shuffle](new SameLengthByteArrayComparator())
 
   val ssf = ssl.getServerSocketFactory()
-  val ss = ssf.createServerSocket(port)
-  ss.asInstanceOf[SSLServerSocket].setNeedClientAuth(true)
+  val ss = ssf.createServerSocket(port).asInstanceOf[SSLServerSocket]
```

Comment:
Why don't you need to `setNeedClientAuth` anymore?

Reply:
When I wrote the local mode shuffler I made it use mutual TLS (mTLS). The services aren't ready for mTLS yet, so I disabled it. I'll restore once mTLS is ready.